shaders/src/light_punctual.fs

```diff
@@ -211,18 +211,15 @@ void evaluatePunctualLights(const MaterialInputs material,
         if (light.NoL > 0.0) {
             if (light.castsShadows) {
                 uint shadowIndex = light.shadowIndex;
-                highp vec4 shadowPosition;
                 if (light.type == LIGHT_TYPE_POINT) {
                     // point-light shadows are sampled from a direction
                     highp vec3 r = getWorldPosition() - light.worldPosition;
-                    highp uint face = 0u;
-                    // getShadowPosition returns zLight which is needed for PCSS/DPCF
-                    shadowPosition = getShadowPosition(r, shadowIndex, light.zLight, face);
+                    highp uint face = getPointLightFace(r);
```

Comment:
Why is the face `highp`? Won't it be `0..5`?

Reply:
good catch, it's an oversight.